runtime/src/lib.rs

```diff
@@ -545,15 +546,25 @@ impl InstanceFilter<Call> for ProxyType {
 					| Call::Reactions(..)
 					| Call::AccountFollows(..)
 					| Call::SpaceFollows(..)
+					| Call::Spaces(..)
+					| Call::Profiles(..)
 			),
 			ProxyType::Management => matches!(
 				c,
-				Call::Spaces(..)
-					| Call::SpaceOwnership(..)
+				Call::SpaceOwnership(..)
```

Comment:
Let's leave it as it for now. We need to rethink this - at least whether we have a further need in it.

```suggestion
				Call::Spaces(..)
					| Call::SpaceOwnership(..)
```